src/design-pages/passwordless_gdm.rst

```diff
@@ -0,0 +1,461 @@
+Passwordless-GDM authentication integration
+=======================================================
+
+Related ticket(s):
+------------------
+ * https://github.com/SSSD/sssd/issues/7069
+
+Problem statement
+-----------------
+Passwordless authentication is becoming increasingly popular. SSSD and FreeIPA
+already provide several authentication mechanisms which make use of it:
+`Smart Cards <https://sssd.io/design-pages/smartcards.html>`__,
+`External Identity Providers
+<https://freeipa.readthedocs.io/en/latest/designs/external-idp/external-idp.html>`__
+(EIdP) and `Passkeys <https://sssd.io/design-pages/passkey_kerberos.html>`__.
+Unfortunately, the integration of these mechanisms into the graphical interface
+leaves much to be desired. Some of them may work in a degraded mode, while
+others can’t be used at all.
+
+SSSD and the GUI should be better integrated to make all these authentication
+mechanisms effortless for the user. This would increase the overall security of
+the system, by providing the benefits of these authentication mechanisms, i.e.
+passwordless, MFA, etc.
+
+SSSD and `GDM <https://wiki.gnome.org/Projects/GDM>`__ are working together to
+provide a set of interfaces that can be used to enable these authentication
+mechanisms in Linux’s GUI. While the initial work targets SSSD-GDM integration,
+the objective is that these interfaces can be used by any other desktop
+environment.
+
+The use of new tools to authenticate users, such as 2FA, U2F and FIDO2, is
+becoming increasingly popular. Currently SSSD provides local authentication of
+a centrally managed user with passkeys, but it doesn't provide any way to
+authenticate remotely to a system. This diminishes the value provided by SSSD
```

Comment:
Hi,

if you want to keep this paragraph (since it is not strictly related to the SSSD GDM integration I think it can be dropped as well) I would like to suggest to mention the FreeIPA Kerberos integration where the Kerberos ticket can be used to connect to remote systems managed by FreeIPA.

bye,
Sumit